servers/rendering/rendering_device.cpp

```diff
@@ -346,7 +346,7 @@ void RenderingDevice::_bind_methods() {
 
 	ClassDB::bind_method(D_METHOD("create_local_device"), &RenderingDevice::create_local_device);
 
-	ClassDB::bind_method(D_METHOD("set_resource_name"), &RenderingDevice::set_resource_name);
+	ClassDB::bind_method(D_METHOD("set_resource_name", "id", "name"), &RenderingDevice::set_resource_name);
```

Comment:
FYI @clayjohn, arguments should be named when binding methods (otherwise they're `arg0` and `arg1`).